src/components/common/TextBox.jsx

```diff
@@ -1,16 +1,28 @@
+import { useState } from "react";
+
 import Button from "./Button";
 
 import { Header } from "../../style/CommonStyle";
 import { TextBoxContainer, TextBoxContent } from "../../style/TextBoxStyle";
 
 const TextBox = ({ title }) => {
+  const [isMinimized, setIsMinimized] = useState(false);
+
+  const handleMinimized = () => {
+    setIsMinimized(!isMinimized);
```

Comment:
`setIsMinizied`를 이용하여 상태를 변경할 때, 아래 코드와 현 코드 중 가독성과 일관성 면에서 더 좋은 코드는 무엇이라고 생각하실까요?
```suggestion
    setIsMinimized(true);
```

Reply:
저는 양래님께서 토글기능을 구현하시려는 의도로 작성하셨다고 생각해요! 초기에 false로 설정했지만, 사용자 상호작용을 통해 state가 true로 설정되는 경우도 있으니까요!

Reply:
기연님께서 잘 말씀해주셨군요!
해당 기능에서는 위와 같이 작성하는 것이 더 유연하게 대처할 수 있을 것 같다고 생각합니다.
상황에 맞게 setState함수의 매개변수를 넣어주면 될 것 같습니다!